valgrind/parse_valgrind_suppressions.sh

```diff
@@ -0,0 +1,58 @@
+#! /usr/bin/awk -f
```

Comment:
OK, suppressing the third party library valgrind problems that we don't want to deal with...